unskript-ctl/templates/check.py.template

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel, Field
+from typing import Optional, Tuple
+import pprint
+
+
+class InputSchema(BaseModel):
+    pass
```

Comment:
Should we add some comments like what could a sample InputSchema be? like what is the format we expect, need to have a Field, need to have a title, description and/or default value. Etc..?